boards.txt

```diff
@@ -187,6 +194,20 @@ esp32h2.menu.EraseFlash.none.upload.erase_cmd=
 esp32h2.menu.EraseFlash.all=Enabled
 esp32h2.menu.EraseFlash.all.upload.erase_cmd=-e
 
+esp32h2.menu.Zigbee.default=Disabled
+esp32h2.menu.Zigbee.default.build.zigbee_mode=
+esp32h2.menu.Zigbee.default.build.extra_libs=
+esp32h2.menu.Zigbee.ed=Zigbee ED (end device)
+esp32h2.menu.Zigbee.ed.build.zigbee_mode=-DZIGBEE_MODE_ED
+esp32h2.menu.Zigbee.ed.build.extra_libs=-lesp_zb_api_ed -lesp_zb_cli_command -lzboss_stack.ed.trace -lzboss_stack.ed -lzboss_port
```

Comment:
let's allow `build.extra_libs` to be available for something else that would be use-case specific. Add `build.zigbee_libs` instead. Makes it more consistent and prevents some other feature to overwrite the variable.

Reply:
Done, good catch as that could collide with the PSRAM extra_libs. Renamed as suggested